css/style.css

```diff
@@ -8,24 +8,46 @@
   --bs-font-sans-serif: 'Poppins', sans-serif !important;
 }
 
-html {
-  background-color: var(--default-bg);
+body {
+  background-color: var(--default-bg)!important;
+  grid-template-columns: 100%;
+  grid-template-rows: 60px auto 90px;
+  grid-template-areas:
+  "header"
+  "content"
+  "footer";
+}
+
+body, a, button, textarea, input {
+  font-family: var(--bs-font-sans-serif);
+  font-size: 14px !important;
+}
+
+a {
+  color: var(--text);
+  text-decoration: none;
 }
 
 /*------------ NavBar-----------------*/
 
-.btn.btn-default {
-  background-color: var(--text-primary);
-  border: 1px solid var(--text-primary);
-  color: var(--text-hl);
+nav.bg-light {
+  background-color: var(--default-bg)!important;
 }
 
-.nav-item .btn.btn-second, .nav-item .btn.btn-default {
-  margin-left: 15px;
+nav {
+  min-height: 60px;
+  grid-area: header;
+}
+
+.nav-link#home, .nav-link#services, .nav-link#projects, .nav-link#contact, .nav-link#gallery, .nav-link#profile {
+  color: var(--text-primary) !important;
+  text-decoration: underline;
```

Comment:
todos os itens do navbar terão underline mesmo? não seria só o início?

Reply:
terá underline os links ativos, por exemplo: eu estou fazendo a página home então eu coloco no arquivo um id home na nav, se eu tiver fazendo a página de serviços eu removo o id da home  e coloco o id nos serviços (dentro da nav), dessa forma vai ficar ativo os links de cada página para que o usuário saiba em qual página esta

![image](https://user-images.githubusercontent.com/94807208/194773086-327241c3-0a16-40ef-96be-0fac4048602f.png)